packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.tsx

```diff
@@ -100,12 +109,25 @@ export class AtomicInsightInterface
 
   public connectedCallback() {
     this.store.setLoadingFlag(FirstInsightRequestExecutedFlag);
+    this.updateFieldsToInclude();
+  }
+
+  private updateFieldsToInclude() {
+    if (this.fieldsToInclude) {
+      this.store.set(
+        'fieldsToInclude',
+        this.fieldsToInclude.split(',').map((field) => field.trim())
+      );
+    }
   }
 
   /**
    * Initializes the connection with the headless insight engine using options for `accessToken` (required), `organizationId` (required), `renewAccessToken`, and `platformUrl`.
    */
   @Method() public initialize(options: InsightInitializationOptions) {
+    if (this.widget) {
+      this.host.setAttribute('widget', 'true');
+    }
```

Comment:
By adding `reflect: true` on `widget`'s `Prop()` you shouldn't need to do this.

Reply:
ah lovely, I didn't realise!!